docs/bclinux-installation-guide.md

```diff
@@ -0,0 +1,8 @@
+# Installing Intel® Clear Containers 3.0 on BCLinux* version 7
+## Introduction of BC-Linux
+[BC-Linux](https://www.bclinux.org/)（BigCloud Enterprise Linux） is an enterprise Linux operating system developed by China Mobile for cloud computing and big data users.
+BC-Linux has developed some common server operating systems based on the CentOS OS Open Source Community Edition. For example, BC-Linux 7.3 is based on the same source code as Red Hat Enterprise RHEL 7.3 and maintains 100% compatibility with RHEL.
+In addition, BCLinux has released some customized versions of server OS for different scenarios. BCLinux-for-DCOS is a representative product of them. It greatly simplifies the common BC-Linux 7 system and integrates the packages required for the container and PaaS Platform. It uses a customized high-version kernel, fixes the security vulnerabilities in the low-version kernel, and adds multiple optimization patches for the container.
+
+## Installation steps
+Clear Containers 3.0 is available for BC-Linux 7.3, 7.2 and BCLinux-for-DCOS-V1.0. For now, BC-Linux is a variant of CentOS in a sense, so users can refer to docs/centos-installation-guide.md to install Clear Containers 3.0 on BCLinux* version 7.
```

Comment:
Clear Containers 3.0 is available for BC-Linux 7.3, 7.2, and BCLinux-for-DCOS-V1.0. BC-Linux is basically a variant of CentOS so users can refer to docs/centos-installiation-guide.md to install Clear Containers 3.0 on BCLinux* version 7.